src/plugins/plot/PlotView.vue

```diff
@@ -197,18 +197,23 @@ export default {
 
       if (this.compositionCollection) {
         this.compositionCollection.on('add', this.subscribeToStaleness);
-        this.compositionCollection.on('remove', this.triggerUnsubscribeFromStaleness);
+        this.compositionCollection.on('remove', this.removeSubscription);
         this.compositionCollection.load();
       }
     },
+    removeSubscription(identifier) {
+      this.triggerUnsubscribeFromStaleness({
```

Comment:
`triggerUnsubscribeFromStaleness` is expecting a domainObject (`{identifier: {namepsace: xyz, key: abc}}`). Composition `remove` event only sends an identifier `{namespace: xyz key: abc}`)

So we need to make sure we're sending in an object instead of just an identifier.